cloud/jenkins/pxc_operator_gke_alpha.groovy

```diff
@@ -0,0 +1,359 @@
+                        ShutdownCluster('basic')
+                    }
+                }
+                stage('E2E Scaling') {
+                    steps {
+                        CreateCluster('scaling')
+                        runTest('scaling', 'scaling')
+                        runTest('scaling-proxysql', 'scaling')
+                        runTest('security-context', 'scaling')
+                        ShutdownCluster('scaling')
+                    }
+                }
+                stage('E2E SelfHealing') {
+                    steps {
+                        CreateCluster('selfhealing')
+                        runTest('storage', 'selfhealing')
+                        runTest('self-healing', 'selfhealing')
+                        runTest('self-healing-advanced', 'selfhealing')
+                        runTest('operator-self-healing', 'selfhealing')
+                        ShutdownCluster('selfhealing')
+                    }
+                }
+                stage('E2E Backups') {
+                    steps {
+                        CreateCluster('backups')
+                        runTest('recreate', 'backups')
+                        runTest('restore-to-encrypted-cluster', 'backups')
+                        runTest('demand-backup', 'backups')
+                        runTest('demand-backup-encrypted-with-tls', 'backups')
+                        runTest('scheduled-backup', 'backups')
+                        runTest('pitr','backups')
+                        ShutdownCluster('backups')
+                    }
+                }
+                stage('E2E BigData') {
+                    steps {
+                        CreateCluster('bigdata')
+                        runTest('big-data', 'bigdata')
+                        ShutdownCluster('bigdata')
+                    }
+                }
+            }
+        }
+    }
+    post {
+        always {
+            makeReport()
+            sh """
+                echo "${TestsReport}" > TestsReport.xml
+            """
+            step([$class: 'JUnitResultArchiver', testResults: '*.xml', healthScaleFactor: 1.0])
+            archiveArtifacts '*.xml'
+
+            withCredentials([string(credentialsId: 'GCP_PROJECT_ID', variable: 'GCP_PROJECT'), file(credentialsId: 'gcloud-alpha-key-file', variable: 'CLIENT_SECRET_FILE')]) {
+                sh '''
+                    export CLUSTER_NAME=$(echo jenkins-pxc-$(git -C source rev-parse --short HEAD) | tr '[:upper:]' '[:lower:]')
+                    source $HOME/google-cloud-sdk/path.bash.inc
+                    gcloud auth activate-service-account alpha-svc-acct@"${GCP_PROJECT}".iam.gserviceaccount.com --key-file=$CLIENT_SECRET_FILE
+                    gcloud config set project $GCP_PROJECT
+                    gcloud alpha container clusters delete --zone us-central1-a $CLUSTER_NAME-basic $CLUSTER_NAME-scaling $CLUSTER_NAME-selfhealing $CLUSTER_NAME-backups $CLUSTER_NAME-bigdata $CLUSTER_NAME-upgrade | true
```

Comment:
please delete the cluster in the following way https://github.com/percona/percona-xtradb-cluster-operator/blob/main/Jenkinsfile#L387

Reply:
https://github.com/Percona-Lab/jenkins-pipelines/pull/1009/commits/b43c55ab86b9ef473cafc63524894ee8c5b9cebd